Vagrantfile

```diff
@@ -157,11 +158,21 @@ scissor = {
     :net_type => NETWORK_TYPE_DHCP,
     :show_gui => false
   },
+  CNIT_CORRELATOR_VM_NAME => {
+    :autostart => false,
+    :box => CENTOS_BOX_ID,
+    :cpus => 1,
+    :mac_address => "0800271F9D11",
+    :mem => 1024,
+    :net_auto_config => false,
+    :net_type => NETWORK_TYPE_DHCP,
+    :show_gui => false
+  },
   PREWIKKA_VM_NAME => {
     :autostart => true,
     :box => CENTOS_BOX_ID,
     :cpus => 1,
-    :mac_address => "0800271F9D11",
+    :mac_address => "0800271F9D12",
```

Comment:
Don't change MAC addresses. They are used to statically assign IP addresses via DHCP.